webtool/llm.py

```
@@ -253,12 +254,28 @@ def _base_url(cfg: dict, prov: dict) -> str:
     return (cfg["base_url"] or prov.get("base", "")).rstrip("/")
 
 
+def _ssl_kontext():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_ssl_kontext`

(ANN202)